pandas/tests/io/test_excel.py

```diff
@@ -1135,7 +1131,7 @@ def test_excel_sheet_size(self):
         with pytest.raises(ValueError, match=msg):
             col_df.to_excel(self.path)
 
-    def test_excel_sheet_by_name_raise(self, *_):
+    def test_excel_sheet_by_name_raise(self, engine, ext):
```

Comment:
The `*_` signature was making practically all fixtures available to the function, so replaced with something explicit (easier now that we've decoupled the class based designed from the fixturized design)

Reply:
> engine and ext do not need to be in the function signature if their values are not required by the test. the parameterisation is handled by the set_engine_and_path autouse fixture.

Good to know. Should definitely be able to simplify some of these methods on next push then!